### DISCUSSION TITLE
How to set semitransparent texture correctly in SimpleMeshLayer

### DISCUSSION THREAD

**Discussion by Emelia-L** [visgl/deck.gl#5427] [Q&A]
When using SimpleMeshLayer,  I have set getColor function, and I want to add a semitransparent texture on the geometry. But after setting texture, only semitransparent texture can show, the color is not effective.
And I set parameters{ blend :true, blendFunc: [GL.SRC_ALPHA, GL.ONE_MINUS_SRC_ALPHA]} for SimpleMeshLayer, it still cannot work.
But when I set highLightColor, when I pick a geometry, highlightColor and semitransparent texture can blend.
I wonder how can I make geometry color and semitransparent texture blend in SimpleMeshLayer.

**Comment by Pessimistress** [comment]
From `getColor` documentation:

> If mesh does not contain vertex colors, use this color to render each object. If mesh contains vertex colors, then the two colors are mixed together. Use [255, 255, 255] to use the original mesh colors. If texture is assigned, then both colors will be ignored.

https://deck.gl/docs/api-reference/mesh-layers/simple-mesh-layer#getcolor

If you just want to add opacity to the texture, use `opacity`.
